extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1783,7 +1790,12 @@ export class AmpStoryPage extends AMP.BaseElement {
 
       this.mutateElement(() => {
         textEl.textContent = openLabel;
-        this.element.appendChild(this.openAttachmentEl_);
+
+        const container = this.win.document.createElement('div');
+        container.classList.add('i-amphtml-page-attachment-host');
+        container.setAttribute("role", "button");
```

Comment:
This block should be outside of the `mutateElement` block, we can reserve `mutateElement` for operations that write in the DOM (`textContent`, `appendChild` and `createShadowRootWithStyle` in this case)